test/OpenTelemetry.Instrumentation.Http.Tests/HttpClientTests.Basic.netcore31.cs

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
-#if NETCOREAPP3_1_OR_GREATER
+#if NET6_0_OR_GREATER
```

Comment:
I guess the name of the file no longer makes sense. One idea:
* rename the file to `HttpClientTests.Basic.cs`
* Change the directive to `#if !NETFRAMEWORK`.